routes/cart.route.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const router = express.Router();
+const Cart = require("../models/cart.model")
+
+// GET
+router.get('/', async(req ,res)=>{
+
+  try{
+    const prd = await Cart.find()
+    res.json(prd)
+  }catch(err){
+    console.log("err", err)
+  }
+
+});
+
+// POST
+router.post('/', async (req, res)=>{
+  console.log(req.body)
+
+  const addPrd = req.body
```

Comment:
Use appropriate name convention - "addProduct" instead of "addPrd"

Reply:
Yes, done that. 
Thank you